src/mlpack/methods/ann/activation_functions/gcu_function.hpp

```diff
@@ -0,0 +1,89 @@
+/**
+ * @file methods/ann/activation_functions/gcu_function.hpp
+ * @author Tarek Elsayed
+ *
+ * Definition and implementation of the Growing Cosine Unit function.
+ *
+ * For more information, see the following paper.
+ *
+ * @code
+ * @misc{
+ *   authors = {Mathew Mithra Noel, Arunkumar L, Advait Trivedi, Praneet Dutta},
+ *   title = {Growing Cosine Unit: A Novel Oscillatory Activation Function That Can Speedup Training and Reduce Parameters in Convolutional Neural Networks},

```

Comment:
Can you make sure this fits into our 80 character line limit; see https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping for more details.

Reply:
Sure